jsonnet/mixin/alerts/alerts.libsonnet

```
@@ -104,6 +104,25 @@
           },
         ],
       },
+      {
+        name: 'config-reloaders',
+        rules: [
+          {
+            alert: 'ConfigReloaderSidecarErrors',
+            expr: |||
+              (min by (job,pod) (reloader_last_reload_successful{job=~"%(prometheusSelector)s|%(alertManagerSelector)s"}) == 0)
```

To deal with missed scrapes, we need an *_over_time() function:

```suggestion
              max_over_time(reloader_last_reload_successful{job=~"%(prometheusSelector)s|%(alertManagerSelector)s"}[5m]) == 0
```